Makefile

```diff
@@ -66,18 +66,20 @@ endif
 
 $(OUT)/trivialDAO_storage.tz : admin_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
 $(OUT)/trivialDAO_storage.tz : token_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
+$(OUT)/trivialDAO_storage.tz : token_id = 0n
```

Comment:
Perhaps we should add `governance` to these options as well, otherwise we risk having some ambiguity (if not now soon), I propose simply, `governance_token_address` and `governance_token_id`.

Reply:
Changed